### PR TITLE
Add FeedbackToken

### PR DESCRIPTION
**Pull request by palango**
This adds the type FeedbackToken which will be used for the feedback
mechanism. It also adds code to save and load it from the database.

Part of #336

### CODE REVIEW

File: src/pathfinding_service/database.py [hunk 1]
```diff
@@ -203,3 +204,27 @@ def delete_channel_views(self, channel_id: ChannelID) -> None:
     def get_token_networks(self) -> Iterator[TokenNetwork]:
         for row in self.conn.execute("SELECT address FROM token_network"):
             yield TokenNetwork(token_network_address=decode_hex(row[0]))
+
+    def insert_feedback_token(self, token: FeedbackToken) -> None:
+        token_dict = dict(token_id=token.id.hex, expiry=token.expiry)
+        self.conn.execute(
+            """
+            INSERT INTO feedback_token (
+                token_id, expiry
```
Comment (karlb) [inline src/pathfinding_service/database.py]: I would rather store the time at which the token is generated. This seems like the more meaningful time to me. Should work either way, though.

File: src/pathfinding_service/schema.sql [hunk 2]
```diff
@@ -46,5 +46,11 @@ CREATE TABLE capacity_update (
     PRIMARY KEY (updating_participant, token_network_address, channel_id)
 );
 
+CREATE TABLE feedback_token (
+    token_id CHAR(32) NOT NULL,
+    expiry TIMESTAMP NOT NULL,
+    PRIMARY KEY (token_id)
+);
+
 CREATE UNIQUE INDEX one_active_session_per_sender
```
Comment (karlb) [inline src/pathfinding_service/schema.sql]: This constraint belongs to the `IOU` table, so better keep it right below it.

File: tests/pathfinding/test_database.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
+from uuid import uuid4
+
+from pathfinding_service.model import FeedbackToken
+
+
+def test_save_and_load_feedback_token(pathfinding_service_mock):
+    token = FeedbackToken(id=uuid4(), expiry=datetime.now())
```
Comment (karlb) [inline tests/pathfinding/test_database.py]: Using `utcnow` usually leads to less problems down the road.

**Comment by palango** [comment]
All comments made a lot of sense and are fixed now.

**Pull request merged**